pkgs/by-name/ol/ollama/package.nix

```diff
@@ -169,14 +169,10 @@ goBuild {
     ++ lib.optionals enableCuda cudaLibs
     ++ lib.optionals stdenv.hostPlatform.isDarwin metalFrameworks;
 
-  patches = [
-    # ollama's build script is unable to find hipcc
-    ./rocm.patch
-  ];
-
```

Comment:
Why was this removed?

Reply:
upstream reorganized the make files 

```
$ cat make/rocm-defs.make

# Common definitions for the various Makefiles which set cuda settings
# No rules are defined here so this is safe to include at the beginning of other makefiles

ifeq ($(OS),windows)
	HIP_COMPILER:=$(wildcard $(HIP_PATH)/bin/hipcc.bin.exe)
else ifeq ($(OS),linux)
	HIP_PATH?=$(shell ls -d /opt/rocm 2>/dev/null)
	HIP_COMPILER:=$(wildcard $(HIP_PATH)/bin/hipcc)
endif
```

I'm not sure if we need to make a new patch yet as I got another error